Source/NSFileManager.m

```diff
@@ -2032,6 +2026,7 @@ - (NSDictionary*) fileSystemAttributesAtPath: (NSString*)path
 
   if (!GetVolumePathNameW(lpath, volumePathName, 128))
     {
+      *error = [NSError _last];
```

Comment:
Oh, nevermind. I just got where the problem is (thanks to @rfm's insight). Thanks for the heads-up! :)